### PR TITLE
Export query window and grid controllers, to be able to extend them

### PR DESCRIPTION
**Pull request by llienher**
I need this to be able to extend a controller in a project.

### CODE REVIEW

File: contribs/gmf/src/query/gridComponent.js [hunk 1]
```diff
@@ -163,13 +163,12 @@ module.component('gmfDisplayquerygrid', queryGridComponent);
...
  * @param {!JQuery} $element Element.
  * @constructor
- * @private
  * @hidden
  * @ngInject
  * @ngdoc controller
  * @ngname GmfDisplayquerygridController
  */
-function Controller($injector, $scope, ngeoQueryResult, ngeoMapQuerent,
+export function Controller($injector, $scope, ngeoQueryResult, ngeoMapQuerent,
```
Comment (sbrunner) [inline contribs/gmf/src/query/gridComponent.js]: Should be named `QueryGridComponent`

File: contribs/gmf/src/query/windowComponent.js [hunk 2]
```diff
@@ -136,13 +136,12 @@ module.component('gmfDisplayquerywindow', queryWindowComponent);
...
  *     overlay manager service.
  * @constructor
- * @private
  * @hidden
  * @ngInject
  * @ngdoc controller
  * @ngname GmfDisplayquerywindowController
  */
-function Controller($element, $scope, ngeoQueryResult, ngeoMapQuerent, ngeoFeatureOverlayMgr) {
+export function Controller($element, $scope, ngeoQueryResult, ngeoMapQuerent, ngeoFeatureOverlayMgr) {
```
Comment (sbrunner) [inline contribs/gmf/src/query/windowComponent.js]: Should be named `QueryWindowComponent`...

Comment (gberaudo) [reply contribs/gmf/src/query/windowComponent.js]: What about QueryWindowController ^^?

Comment (sbrunner) [reply contribs/gmf/src/query/windowComponent.js]: :-)

Comment (llienher) [reply contribs/gmf/src/query/windowComponent.js]: Done in this PR: https://github.com/camptocamp/ngeo/pull/4775

**Pull request merged**